debugging/tasks/analyze.js

```diff
@@ -521,8 +520,71 @@ exports.run = function (extra, args) {
     }
   };
 
+  const extractUnplannedFailoverCandidates = (info, dump) => {
+    const planCollections = dump.arango.Plan.Collections;
+    const currentCollections = dump.arango.Current.Collections;
+    const fixes = [];
+    Object.keys(currentCollections).forEach(function (dbname) {
```

Comment:
Just FYI:

This can be Written as
```
for (const [dbname, database] of Object.entries(currentCollections)) {
```
i tend to think this is more readable, but both variants are correct.

Reply:
we adjust

Reply:
@mchacki please check